src/Illuminate/Encryption/EncryptionServiceProvider.php

```diff
@@ -37,8 +39,28 @@ protected function registerEncrypter()
      * Configure Opis Closure signing for security.
      *
      * @return void
+     *
+     * @deprecated It will be removed in Laravel 9.
      */
     protected function registerOpisSecurityKey()
+    {
+        if (\PHP_VERSION_ID < 80100) {
+            $config = $this->app->make('config')->get('app');
+
+            if (! class_exists(OpisSerializableClosure::class) || empty($config['key'])) {
```

Comment:
no need for this class exists check. it will always exist

Reply:
Well, `illuminate/encryption` may be used as isolated package. So I rather not touch that code.